zerver/lib/notifications.py

```diff
@@ -364,6 +392,7 @@ def handle_missedmessage_emails(user_profile_id, missed_email_events):
 
 @uses_mandrill
 def clear_followup_emails_queue(email, mail_client=None):
+    # type: (django_models.EmailField, Optional[mandrill.Mandrill]) -> None
```

Comment:
`email` is just a str (EmailField objects are just a str with some validation done on it)